pkgs/development/python-modules/pytmx/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, pygame, pyglet, pysdl2, six }:
+
+buildPythonPackage rec {
+  pname = "pytmx";
+  version = "3.21.7";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "PyTMX";
+    sha256 = "19wgjdhkzzi2kpnba6xpr29b24dqy3nhdvcfardfm738mzxc3sgn";
+  };
+
+  propagatedBuildInputs = [ pygame pyglet pysdl2 six ];
+
+  meta = {
```

just a personal preference, but what I commonly see:
```suggestion
  meta = with lib; {
```